index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <section id='welcome-section'>
+      <h1>Personal Portfolio</h1>
+    </secton>
+    <section id='projects'>
+      <a href='www.wikipedia.com'>link</a>
+    </section>
```

Comment:
Please wrap this in a main tag to improve the semantic quality of the page.

Reply:
[OPTIONAL] I suggest you include a link or a better explanation of what semantic tags are to make sure he collaboarator knows what exactly you mean. 

an example for this could be:
Please wrap this in a main tag to improve the semantic quality of the page. See this [link](https://developer.mozilla.org/en-US/docs/Glossary/Semantics#semantics_in_html) to understand html semantic tags better.